### PR TITLE
Model Refactor

### PR DESCRIPTION
**Pull request by AHABID**
NGSI-LD schema for all entities

demandeCategory  & sourceCategory properties for Junction, Reservoir and Tank.

### CODE REVIEW

**Comment by bobeal** [comment]
You have some weird indentations that make the files hard to read

File: Junction/example-normalized-ld.jsonld [hunk 1]
```diff
@@ -1,70 +1,378 @@
...
-    "createdAt": "2020-02-20T15:42:00Z",
-    "modifiedAt": "2020-02-20T15:45:00Z",
-    "location": {
-        "type": "GeoProperty",
-        "value": {
-            "type": "Point",
-            "coordinates": [
-               24.30623,
-               60.07966
+    "$schema": "http://json-schema.org/schema#",
```
Comment (bobeal) [inline Junction/example-normalized-ld.jsonld]: this file should contain an NGSI-LD example, not the JSON schema

**Comment by bobeal** [comment]
Overall, it seems good.

File: Junction/example-normalized-ld.jsonld (hunk 1 above)
Comment (AHABID) [reply Junction/example-normalized-ld.jsonld]: Done :D

**Pull request merged**